File: problems/running_sum_of_1d_array/solution.go
func runningSum(nums []int) []int {
    result:=make([]int,0,len(nums))
    sum:=0
    for i:=0;i<len(nums);i++{
        sum+=nums[i]
        result=append(result,sum)
    }
    
    return result
}

File: problems/running_sum_of_1d_array/solution.py
class Solution:
    def runningSum(self, nums: List[int]) -> List[int]:
        result = [0] * len(nums)
        
        prev = 0
        for i in range(len(nums)):
            result[i] = prev + nums[i]
            prev = result[i]
            
        return result